react/components/SKUSelector/utils/index.ts

```diff
@@ -179,3 +179,50 @@ const buildAvailableVariations = (
   }
   return result
 }
+
+export const DEFAULT_WIDTH = 'auto'
```

Comment:
sidenote, specially to @lbebber :

Esse código ta presente me umas 4 apps diferentes, talvez seja interessante criar uma app store-framework utils que coloca isso e outras coisas la.

Talvez uma app no NPM com os types do ProductContext tbm, etc